modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/FDSEntries.tsx

```diff
@@ -794,6 +799,115 @@ const FDSEntries = ({
 		});
 	};
 
+	const RenameFDSEntryModalContent = ({
+		closeModal,
+		itemData,
+		loadData,
+		namespace,
+	}: {
+		closeModal: Function;
+		itemData: FDSEntryType;
+		loadData: Function;
+		namespace: string;
+	}) => {
+		const [fdsEntryNameValue, setFdsEntryNameValue] = useState('');
```

Comment:
Initial value here should be the existing name.

In addition:
- `setFDS*`
- Why value in `NameValue`? Why not just `Name`? 
- It's better to name this be `label` and not `name`, like [the same element in add modal](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/FDSEntries.tsx#L585),  and the field in persistence. `name` has implication that it is unique identifier, while `label` is clearly just for UI, which is the case here.